services/static-webserver/client/source/class/osparc/editor/ThumbnailSuggestions.js

```diff
@@ -49,17 +49,31 @@ qx.Class.define("osparc.editor.ThumbnailSuggestions", {
   },
 
   statics: {
+    defaultTemplates: [
```

Comment:
It looks very very nice! 🤩 

Ah, so you're basically hard-coding those urls here, if I am understanding this well. If this is true, would it be hard to get "dynamically" the list of thumbnails from a certain url (e.g. "https://raw.githubusercontent.com/ZurichMedTech/s4l-assets/main/app/full/project_thumbnails/? 

Related to the point above: will all products get this list of thumbnails? Since these thumbnails are specific to s4l, I don't think it's a good idea to have them in `tis` or `osparc` products.

Reply:
Just a template till we get the proper list 🤣

Reply:
Aaah, I see. 😅  And where will the proper list go, in the DB?
Can the list be displayed/customized by product?

Reply:
My plan to finish up will be number the images 1 to 10 and they would be added to github assets. I can then make a dir for each product allowing us to have different ones for each